### PR TITLE
feat: Added Scanner. Done.

### PR DESCRIPTION
**Pull request by lupugabriel**

### CODE REVIEW

File: MobileWallet/Screens/Scan/ScanViewController.swift [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+        return true
+    }
+
+    override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
+        return .portrait
+    }
+
+    // MARK: - Private functions
+    private func customizeViews() {
+        topLeftWhiteView.addTopBorder(with: .white, andWidth: 11)
```
Comment (Jasonvdb) [inline MobileWallet/Screens/Scan/ScanViewController.swift]: Move the color into the `Theme.swift` file. Don't want to have any hardcoded colors in the app so it's easier to tweak designs globally later especially when we implement dark mode.

**Comment by Jasonvdb** [comment]
Don't hard code text on the story board, allow for the VC that calls on the scanning VC to set the display text. So we can allow for the scanning component to be reused in the future by other parts of the app and we can use `NSLocalizedString` for later translations

File: MobileWallet.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1439,7 +1463,7 @@
 				CLANG_CXX_LIBRARY = "libc++";
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 8;
-				DEVELOPMENT_TEAM = 8974PBB98U;
+				DEVELOPMENT_TEAM = N4J2L6U84U;
```
[the remark this answers is not in the document]
Comment (lupugabriel) [reply MobileWallet.xcodeproj/project.pbxproj]: Sorry about that. i added also a file in the project and i forgot i also changed the team.

File: MobileWallet/Common/Extensions/UIViewExtension.swift [hunk 3]
```diff
@@ -0,0 +1,75 @@
+//  UIViewExtension.swift
```
[the remark this answers is not in the document]
Comment (lupugabriel) [reply MobileWallet/Common/Extensions/UIViewExtension.swift]: Done

File: MobileWallet/Info.plist [hunk 4]
```diff
@@ -104,6 +104,8 @@
 	</array>
 	<key>UIUserInterfaceStyle</key>
 	<string>Light</string>
+	<key>NSCameraUsageDescription</key>
+	<string>We need camera to scan</string>
```
[the remark this answers is not in the document]
Comment (lupugabriel) [reply MobileWallet/Info.plist]: Done

File: MobileWallet/Screens/Scan/ScanViewController.swift [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+        fillLayer.fillRule = .evenOdd
+        fillLayer.fillColor = UIColor.black.cgColor
+        fillLayer.opacity = 0.7
+        view.layer.insertSublayer(fillLayer, below: backButton.layer)
+
+        captureSession.startRunning()
+    }
+
+    private func failed() {
+        let ac = UIAlertController(title: "Scanning not supported", message: "Your device does not support scanning a code from an item. Please use a device with a camera.", preferredStyle: .alert)
```
[the remark this answers is not in the document]
Comment (lupugabriel) [reply MobileWallet/Screens/Scan/ScanViewController.swift]: Ok, i will not change anything here. thanks for feedback

File: MobileWallet/Screens/Scan/ScanViewController.swift (hunk 1 above)
Comment (lupugabriel) [reply MobileWallet/Screens/Scan/ScanViewController.swift]: Done

**Pull request merged**